builtin/providers/aws/resource_aws_dhcp_option_set.go

```
@@ -0,0 +1,98 @@
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"domain_name_servers": &schema.Schema{
+						Type:        schema.TypeString,
+						Required:    true,
+						RequiresNew: true,
+					},
+
+					"domain_name": &schema.Schema{
+						Type:        schema.TypeString,
+						Required:    true,
+						RequiresNew: true,
+					},
+
+					"ntp_servers": &schema.Schema{
+						Type:        schema.TypeString,
+						Required:    true,
+						RequiresNew: true,
+					},
+
+					"netbios_name_servers": &schema.Schema{
+						Type:        schema.TypeString,
+						Required:    true,
+						RequiresNew: true,
+					},
+
+					"netbios_node_type": &schema.Schema{
+						Type:        schema.TypeString,
+						Required:    true,
+						RequiresNew: true,
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourceAwsDhcpOptionSetCreate(d *schema.ResourceData, meta interface{}) error {
+	// Create DHCP Options set options
+	createDhcpOpts := &ec2.CreateDhcpOptions{
+		DomainNameServers:  d.Get("domain_name_servers").(string),
+		DomainName:         d.Get("domain_name").(string),
+		NtpServers:         d.Get("ntp_servers").(string),
+		NetbiosNameServers: d.Get("netbios_name_servers").(string),
+		NetbiosNodeType:    d.Get("netbios_node_type").(string),
+	}
+
+	// Create the DHCP Options set
+	log.Printf("[DEBUG] DHCP Options create config: %#v", createDhcpOpts)
+	dhcpResp, err := ec2conn.CreateDhcpOptions(createDhcpOpts)
+	if err != nil {
+		return fmt.Errorf("Error creating DHCP Options: %s", err)
+	}
+
+	// Get the ID
+	dhcp := &dhcpResp.DhcpOptions
+	log.Printf("[INFO] DHCP Options Set ID: %s", dhcp.DhcpOptionsId)
+	d.SetId(dhcp.DhcpOptionsId)
+
+	return nil
+}
+
+func resourceAwsDhcpOptionSetDelete(d *schema.ResourceData, meta interface{}) error {
+	ec2conn := meta.(*AWSClient).ec2conn
+
+	log.Printf("[INFO] Deleting DHCP Options Set: %s", d.Id())
+
+	if _, err := ec2conn.DeleteDhcpOptions(d.Id()); err != nil {
+		ec2err, ok := err.(*ec2.Error)
+		if ok {
```

Not sure I get what is happening here? Why do we ignore this error?